File: service/recite-words/src/main/java/org/noodle/bean/exercise/ExerciseVocabularyDto.java
package org.noodle.bean.exercise;

/**
 * 个人可测验词汇
 * @author wanghy
 */
public class ExerciseVocabularyDto {
    /**
     * 词汇id
     */
    private Long vocabularyId;

    /**
     * 拼写
     */
    private String spelling;

    /**
     * 发音
     */
    private String pronunciation;

    /**
     * 释义
     */
    private String explanation;

    public Long getVocabularyId() {
        return vocabularyId;
    }

    public void setVocabularyId(Long vocabularyId) {
        this.vocabularyId = vocabularyId;
    }

    public String getSpelling() {
        return spelling;
    }

    public void setSpelling(String spelling) {
        this.spelling = spelling;
    }

    public String getPronunciation() {
        return pronunciation;
    }

    public void setPronunciation(String pronunciation) {
        this.pronunciation = pronunciation;
    }

    public String getExplanation() {
        return explanation;
    }

    public void setExplanation(String explanation) {
        this.explanation = explanation;
    }
}

File: client/ReciteWords/sdk/exercise/ExerciseVocabularyDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace sdk.exercise
{
    /// <summary>
    /// 个人可测验词汇
    /// </summary>
    public class ExerciseVocabularyDto
    {
        /// <summary>
        /// 词汇id
        /// </summary>
        public long VocabularyId { get; set; }
        /// <summary>
        /// 拼写
        /// </summary>
        public string Spelling { get; set; }

        /// <summary>
        /// 发音
        /// </summary>
        public string Pronunciation { get; set; }

        /// <summary>
        /// 释义
        /// </summary>
        public string Explanation { get; set; }
    }
}
